### PR TITLE
New Rule: Callback Phishing Invoice from < 30d domains

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

**Comment by morriscode** [comment]
No hits in testing in the last 2 days, this was in response to reported FN's so I don't expect high volume, but this works when the very specific situation is encountered. In terms of FP's I'd expect the risk to be low.

File: detection-rules/bec_invoice_fraud_30d_domains.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+             strings.icontains(.scan.ocr.raw, "lifelock"),
+             strings.icontains(.scan.ocr.raw, "best buy"),
+             strings.icontains(.scan.ocr.raw, "mcafee"),
+             strings.icontains(.scan.ocr.raw, "norton"),
+             strings.icontains(.scan.ocr.raw, "ebay"),
+             strings.icontains(.scan.ocr.raw, "paypal"),
+             strings.icontains(.scan.ocr.raw, "virus"),
+           )
+    )
+  )
```
Comment (jkamdjou) [inline detection-rules/bec_invoice_fraud_30d_domains.yml:60]: let's add unsolicited + no FPs?

File: detection-rules/bec_invoice_fraud_30d_domains.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+attack_types:
+  - "BEC/Fraud"
```
Comment (jkamdjou) [inline detection-rules/bec_invoice_fraud_30d_domains.yml:62]: just confirming, is this not callback phishing given the brands were looking for?

File: detection-rules/bec_invoice_fraud_30d_domains.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+name: "BEC/Fraud - Invoice from sender, reply-to or return-path domain less than 30d old."
```
Comment (jkamdjou) [inline detection-rules/bec_invoice_fraud_30d_domains.yml:1]:
```suggestion
name: "BEC/Fraud: Branded invoice from sender/reply-to domain less than 30 days old"
```

File: detection-rules/bec_invoice_fraud_30d_domains.yml (hunk 1 above)
Comment (jkamdjou) [reply detection-rules/bec_invoice_fraud_30d_domains.yml]: @morriscode this is resolved but i don't see it in the rule, did you mean to do it or nah?

**Review by jkamdjou — APPROVED** [review]

File: detection-rules/bec_invoice_fraud_30d_domains.yml (hunk 1 above)
Comment (morriscode) [reply detection-rules/bec_invoice_fraud_30d_domains.yml]: You right. Should be actually resolved now.

**Comment by morriscode** [comment]
Testing on this looks good, only a single hit.

**Pull request merged**